setup.py

```diff
@@ -64,6 +64,8 @@
 from setuptools import setup
 
 # Our own imports
+sys.path.insert(0, ".")
```

Comment:
Please correct me if I'm wrong but I'm pretty sure distutils/setuptools won't work unless CWD is actually the repository root.

Reply:
The actual assumption is "CWD contains the files required for the build". It doesn't tell you _what else_ does it contain.
While this is not a likely scenario this could fail on, I got burnt enough times by `sys.path.insert`, that I'm (overly, perhaps) pedantic about it. The linter in my brain is screaming when it sees this string of characters.

I'm not going to block the PR because of this, but I invite you to try and find a better way of achieving the right imports. I'm not familiar with `backend-path` - is this something that could substitute for `sys.path.insert`?

Reply:
Ok, then let's leave it as it is. I think I'm being too nitpicky about this anyway.

Reverting path after imports actually makes things worse: the import tree is done, but you have no way to reproduce it by looking into `sys.path` via debugger. For any level lower than `setup.py`, you'd essentially have to _know_ that `sys.path` has been modified.